src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -80,12 +80,17 @@ def __init__(self, *args, **kwargs):
         self.uuids = []
         self.dfuse_hosts = None
         self.num_run_datamover = 0  # Number of times run_datamover was called
+        self.job_manager = None
+        self.parent = None
```

Comment:
[MINOR] For future pr can we change this to something like `self.parent_dir` just to be self explanatory what this variable is for.

Reply:
Yes, agree! I'll need to make some changes for onsite testing, so I'll probably do it there